packages/web-framework/bin/static/scripts/ti-framework.css

```
@@ -2528,4 +2566,383 @@ body {
+.ti-chart[data-ti-chart-type="radar"] svg   { max-width: 360px; }
+.ti-chart[data-ti-chart-type="line"] svg    { max-width: 640px; }
+
+/* Visually-hidden a11y mirror table (screen-reader only). */
+.ti-chart-sr {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.13.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)